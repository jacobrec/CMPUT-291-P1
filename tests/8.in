1
jacob
password
2





0
18
2
2


blue


2
Suzuki




2




IAM-SPD
2

Accord



2



hkasd
fsadfldfsalvja
2017

3
2

