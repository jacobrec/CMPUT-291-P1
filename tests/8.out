Please sign in to use App:
1) login
2) exit
Username: Password: Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Plate: │  Row   │     Make      │     Model     │     Color     │ Year │  Plate  │
╞════════╪═══════════════╪═══════════════╪═══════════════╪══════╪═════════╡
│   1    │     Zoom      │     Fast      │      Red      │ 2017 │ IAM-SPD │
│   2    │    Suzuki     │     Aerio     │     Blue      │ 2006 │ BLU-100 │
│   3    │    Suzuki     │     Swift     │    Yellow     │ 2008 │  NULL   │
│   4    │    Toyota     │     Echo      │      Red      │ 2003 │ WCC-189 │
│   5    │     Honda     │    Accord     │     Beige     │ 2004 │ CHO-099 │
│   6    │   Chevrolet   │    Camaro     │      red      │ 1969 │   P1    │
│   7    │   Chevrolet   │    Camaro     │    yellow     │ 1969 │   P3    │
│   8    │    Toyota     │      MR2      │     white     │ 1992 │   P10   │
│   9    │     Mazda     │     Miata     │      red      │ 1989 │   P2    │
│   10   │     Mazda     │      RX7      │     white     │ 1989 │   P9    │
│   11   │     Jeep      │   Wrangler    │     black     │ 1988 │   P7    │
│   12   │     Honda     │     Civic     │      red      │ 1989 │   P13   │
│   13   │  Mitsubishi   │    Eclipse    │      red      │ 1989 │   P8    │
│   14   │     Lotus     │     Elise     │    orange     │ 1996 │   P11   │
│   15   │     Lotus     │    Esprit     │     blue      │ 1996 │   P14   │
│   16   │   Delorean    │    DMC-12     │    silver     │ 1981 │   P6    │
│   17   │     Dodge     │    Caravan    │      red      │ 1984 │   P4    │
Select a row to get more info: Please enter a number from 1 to 17
Select a row to get more info: Please enter a number from 1 to 17
Select a row to get more info: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│    Suzuki     │     Aerio     │     Blue      │ 2006 │ BLU-100 │     2017-08-09     │     2020-08-09     │     Jacob     │   Reckhard    │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Plate: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│    Suzuki     │     Aerio     │     Blue      │ 2006 │ BLU-100 │     2017-08-09     │     2020-08-09     │     Jacob     │   Reckhard    │
│     Lotus     │    Esprit     │     blue      │ 1996 │   P14   │     2018-12-12     │     2030-08-13     │      Dr.      │      Fox      │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Plate: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│    Suzuki     │     Aerio     │     Blue      │ 2006 │ BLU-100 │     2017-08-09     │     2020-08-09     │     Jacob     │   Reckhard    │
│    Suzuki     │     Swift     │    Yellow     │ 2008 │  NULL   │     2017-01-08     │     2020-01-08     │     Arun      │    Navras     │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Plate: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│     Zoom      │     Fast      │      Red      │ 2017 │ IAM-SPD │     2017-08-08     │     2020-08-08     │    Speedy     │  McSpeedFace  │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Plate: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│     Honda     │    Accord     │     Beige     │ 2004 │ CHO-099 │     2017-06-06     │     2020-06-06     │     Peter     │   \\newline   │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Make: Model: Color: Year: Invalid input for type: number-ish
Year: Invalid input for type: number-ish
Year: Plate: │     Make      │     Model     │     Color     │ Year │  Plate  │ Registration Date  │    Expiry Date     │  First Name   │   Last Name   │
╞═══════════════╪═══════════════╪═══════════════╪══════╪═════════╪════════════════════╪════════════════════╪═══════════════╪═══════════════╡
│     Zoom      │     Fast      │      Red      │ 2017 │ IAM-SPD │     2017-08-08     │     2020-08-08     │    Speedy     │  McSpeedFace  │
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Please sign in to use App:
1) login
2) exit
